std_msgs/Header header
geometry_msgs/Wrench offset
float64 mass
geometry_msgs/Vector3 center_of_mass
float64 score1
float64 score2